scheduler/src/cook/mesos/util.clj

```diff
@@ -198,6 +198,39 @@
      get-pending-jobs-duration
      (get-pending-job-ents* unfiltered-db true))))
 
+(defn generate-intervals
```

Comment:
Can this be replaced with http://clj-time.github.io/clj-time/doc/clj-time.periodic.html#var-periodic-seq ?